README.md

```diff
@@ -43,37 +43,39 @@ The SDK runs on devices with iOS 6.0 or higher.
 	- `SystemConfiguration`
 	- `UIKit`
 	- `CoreTelephony`(only required if iOS > 7.0)
-9. Open the info.plist of your app target and add a new field of type *String*. Name it `MSAIInstrumentationKey` and set your AppInsights instrumentation key as its value.
+9. Open the info.plist of your app target and add a new field of type *String*. Name it `MSAIInstrumentationKey` and set your AppInsights instrumentation key as its value. If you don't have one yet, we also have a guide on [how to get an instrumentation key](https://github.com/Microsoft/AppInsights-Home/wiki#getting-an-application-insights-instrumentation-key)
 
 <a id="modify"></a> 
 ## Modify Code 
 
 ### Objective-C
 
-2. Open your `AppDelegate.m` file.
-3. Add the following line at the top of the file below your own #import statements:
-
+1. Open your `AppDelegate.m` file.
+2. Add the following line at the top of the file below your own #import statements:
 	```objectivec
 	#import <AppInsights/AppInsights.h>
 	```
-4. Search for the method `application:didFinishLaunchingWithOptions:`
-5. Add the following lines to setup and start the AppInsights SDK:
 
+3. Search for the method `application:didFinishLaunchingWithOptions:`
+
+4. Add the following lines to setup and start the AppInsights SDK:
 	```objectivec
-	[[MSAIAppInsights sharedInstance] setup];
-	// Do some additional configuration if needed here
-	...
-	[[MSAIAppInsights sharedInstance] start];
+	[MSAIAppInsights setup];
+	[MSAIAppInsights start];
 	```
 
-	You can also use the following shortcut:
+	If you want to do some additional configuration you have to call the sharedInstance directly between the `setup()` and `start()` calls:
```

Comment:
Change to: 
"If additional configuration is needed, the sharedInstance can be accessed between the `setup()` and `start()` calls: